### PR TITLE
Accept non-pointer values as argument for the Row.WriteSlice method

### PR DESCRIPTION
**Pull request by ninedraft**
There is no need to use pointers to the slices since slices a basically fat pointers by themselfes. 
I propose to permit just plain slices as Row.WriteSlice parameter without breaking API.

### CODE REVIEW

File: write.go [hunk 1]
```diff
@@ -7,32 +7,29 @@ import (
 	"time"
 )
 
-// Writes an array to row r. Accepts a pointer to array type 'e',
+// Writes a slice to row r. Accepts slices and pointers to the slices,
```
Comment (tealeg) [inline write.go:10]:
```suggestion
// Writes a slice to row r. Accepts a slice or a pointer to a slice,
```

File: write.go [hunk 2]
```diff
@@ -7,32 +7,29 @@ import (
...
 // and writes the number of columns to write, 'cols'. If 'cols' is < 0,
```
Comment (tealeg) [inline write.go:11]:
```suggestion
// and will wirte up to the provided number of columns, 'cols'.  If 'cols' is < 0,
```

Comment (tealeg) [reply write.go:11]: I know you didn't add this comment, but it'd be nice to have it corrected.

**Comment by ninedraft** [comment]
done

**Comment by tealeg** [comment]
@ninedraft thanks very much!

**Pull request merged**